client/components/AddItem.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+function AddItem (){
+  return (
+    <div className='AddItem'>
+      <form>
+        <h2>Create New Task</h2>
+        Task:<input type="text" name="taskName" /><br />
+        Due Date:<input type="text" name="due" /><br />
+        Description:<input type="text" name="description" /><br />
+        <button>Submit</button>
```

Comment:
you might want to use the html5 date type - https://www.w3schools.com/html/html_form_input_types.asp